test/executor.test.js

```diff
@@ -52,6 +52,31 @@ describe('executor', function() {
     }
   });
 
+  describe('when booting', function() {
+    it('should set the booting status', function(done) {
+      expect(app.booting).to.be.undefined();
+      boot.execute(app, dummyInstructions, function(err) {
+        expect(err).to.be.undefined();
+        expect(app.booting).to.be.false();
+        done();
+      });
+    });
+
+    it('should emit the `booted` event', function(done) {
+      app.on('booted', function() {
+        expect(true).to.be.ok();
```

Comment:
This assertion never fails, please remove.

If you feel a test without an assertion is confusing, then add a comment at L67 explaining that the test fails on timeout when the event is not emitted.